website/content/en/docs/Config/Port/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Port Forwarding
+weight: 50
+---
+
+Lima supports automatic port-forwarding of ports from guest to host.
```

Comment:
Can we explain that this only applies to localhost ports

Reply:
Done.